src/_packages/dartpad_picker/web/dartpad_picker_main.dart

```diff
@@ -89,7 +89,7 @@ Future<void> main() async {
 
 '''
     .trim();
-var spinning_logo = r'''
+var spinningLogo = r'''
```

Comment:
could this be const?

Reply:
or final?

Reply:
Let's try `const` and see what the analyzer thinks

Reply:
I'm not sure this code is analyzed by the CI, but believe it would fail due to the trailing [.trim()](https://github.com/flutter/website/blob/5fbac81ff12b46ff0d1aa7b965775cb1e79bd45d/src/_packages/dartpad_picker/web/dartpad_picker_main.dart#L168) on each string.

Reply:
Okiday, now it's `final`